java/flight/src/test/java/org/apache/arrow/flight/auth/TestBasicAuth.java

```diff
@@ -79,20 +78,23 @@ public void asyncCall() {
 
   @Test
   public void invalidAuth() {
-    assertThrows(StatusRuntimeException.class, () -> {
+    FlightRuntimeException ex = assertThrows(FlightRuntimeException.class, () -> {
       client.authenticateBasic(USERNAME, "WRONG");
-    }, PERMISSION_DENIED);
+    });
+    Assert.assertEquals(FlightStatusCode.UNAUTHENTICATED, ex.status().code());
 
-    assertThrows(StatusRuntimeException.class, () -> {
-      client.listFlights(Criteria.ALL);
-    }, PERMISSION_DENIED);
+    ex = assertThrows(FlightRuntimeException.class, () -> {
```

Comment:
it might pay to making a "testing" utility class that combines these to lines together.

Reply:
Good idea, thanks.